### DISCUSSION TITLE
CPP code

### DISCUSSION THREAD

**Discussion by jerrypcl** [snakers4/silero-vad#378] [General]
I am looking cpp code  on pytorch instead of onnx for inferencing the vad model.   Where could I get it ? thank you!
Jerry

**Comment by snakers4** [comment]
`/examples/cpp`

It is a community submitted example, use it at your own peril.

**Comment by jerrypcl** [comment]
noted and thanks.  Any c++ samples using pytorch for inferencing?

**Reply by snakers4** [reply]
Not that I know of
In any case, ONNX is 3-4x faster for this particular VAD with python
